api/channel.go

```diff
@@ -877,6 +885,8 @@ func updateLastViewedAt(c *Context, w http.ResponseWriter, r *http.Request) {
 		active = true
 	}
 
+	channelName := data["channel_name"].(string)
```

Comment:
Oh yes you are right, but then I was doing it with the channel name and not the id cause you don't have the channels in the client for the teams you are not currently in, so that means that we are going to need to make a request if you don't have that channel somehow.

any other thoughts on this?